prov/shm/src/smr_dsa.c

```diff
@@ -642,6 +679,154 @@ static bool dsa_check_cmd_status(struct smr_dsa_context *dsa_context,
 }
 
 /* SMR functions */
+
+void smr_dsa_init(void)
+{
+	if (dsa_init)
```

Comment:
How does this get called more than once?

Reply:
It shouldn't but I added it just in case/if the code were to be removed from shm and used more generally like the HMEM stuff but I can remove it. It isn't necessary